spec/models/initiative_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: initiatives
+#
+#  id          :integer          not null, primary key
+#  description :string
+#  startdate   :date
+#  status      :integer
+#  title       :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  parent_id   :integer
+#
+# Indexes
+#
+#  index_initiatives_on_parent_id  (parent_id)
+#
+# Foreign Keys
+#
+#  parent_id  (parent_id => initiatives.id)
+#
+require "rails_helper"
+
+RSpec.describe Initiative, type: :model do
+  let(:initiative) { create(:initiative) }
+
+  it "wires up as an initiative of its source" do
+    expect(initiative.source.initiatives).to include(initiative)
+  end
+
+  it "wires up as an initiative of its helpers" do
+    expect(initiative.helpers.first.initiatives).to include(initiative)
```

Comment:
if you want to do something more dynamic you could use `.sample` instead of `.first`, or just validate that all helpers initiatives include the initiative.

Reply:
Oh I love that! Done!